Lib/idlelib/calltip.py

```diff
@@ -155,6 +155,7 @@ def get_argspec(ob):
     lines = (textwrap.wrap(argspec, _MAX_COLS, subsequent_indent=_INDENT)
              if len(argspec) > _MAX_COLS else [argspec] if argspec else [])
 
+    print(lines)
```

Comment:
```suggestion
```

Reply:
When I hit Commit suggestion, GitHub said "Not authorized to edit …"  even though I am, and did with pencil icon.  I gave them feedback.